packages/salesforcedx-vscode-apex-debugger/src/messages/i18n.ts

```diff
@@ -16,8 +16,14 @@
  * If ommitted, we will assume _message.
  */
 export const messages = {
+  command_error_help_text: 'Command returned the following error',
```

Comment:
Here are some screenshots of messages:

![nosfdxproject](https://user-images.githubusercontent.com/7286437/29228538-f5726a28-7e8e-11e7-9414-852f18699818.png)

![streaminghandshakeerror](https://user-images.githubusercontent.com/7286437/29228545-fdd12178-7e8e-11e7-9fc1-25564c40294d.png)

![startstopsession](https://user-images.githubusercontent.com/7286437/29228551-06723538-7e8f-11e7-807e-69b7ceb20caf.png)